securedrop/i18n.py

```diff
@@ -0,0 +1,76 @@
+#
+# SecureDrop whistleblower submission system
+# Copyright (C) 2017 Loic Dachary <loic@dachary.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+from flask import request, session
+from flask_babel import Babel
+from babel import core
+
+import config
+import os
+
+LOCALES = set(['en_US'])
+babel = None
+
+
+def setup_app(app):
+    global babel
+    babel = Babel(app)
+    assert 1 == len(list(babel.translation_directories))
```

Comment:
Pedantic: Yoda assertion (`expected == test_case` and not `test_case == expected`)

Reply:
The **1** is indeed what is expected. Am I missing something ?

Reply:
I think traditionally it's written `some_operation_or_var == constant`.

Reply:
ah, ok, will change that